migrations/1720788930023_add-generated-reports-table.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Does this need to be run manually with `npm run migrate` or does it run automatically on restart?